### ISSUE TITLE
fails to build with older autotools (${am__api_version}<1.16 && ${am__api_version} != 1.15)

### ISSUE THREAD

**Issue by fullmetalcoding** [pure-data/pure-data#1872]
**Steps to recreate:**
1.) Clone latest release (as of this report 0.53-0)
2.) Per instructions, attempt autogen.sh/configure.sh/make
3.) Observe failure on compiling extras

```
Making all in src
make[2]: Entering directory `/home/inari/pd-0.53-0/src'
Makefile:1146: ../extra/bob~/.deps/libpd_la-bob~.Plo: No such file or directory
Makefile:1147: ../extra/bonk~/.deps/libpd_la-bonk~.Plo: No such file or directory
Makefile:1148: ../extra/choice/.deps/libpd_la-choice.Plo: No such file or directory
Makefile:1149: ../extra/fiddle~/.deps/libpd_la-fiddle~.Plo: No such file or directory
Makefile:1150: ../extra/loop~/.deps/libpd_la-loop~.Plo: No such file or directory
Makefile:1151: ../extra/lrshift~/.deps/libpd_la-lrshift~.Plo: No such file or directory
Makefile:1152: ../extra/pd~/.deps/libpd_la-pdsched.Plo: No such file or directory
Makefile:1153: ../extra/pd~/.deps/libpd_la-pd~.Plo: No such file or directory
Makefile:1154: ../extra/pique/.deps/libpd_la-pique.Plo: No such file or directory
Makefile:1155: ../extra/sigmund~/.deps/libpd_la-sigmund~.Plo: No such file or directory
Makefile:1156: ../extra/stdout/.deps/libpd_la-stdout.Plo: No such file or directory
make[2]: *** No rule to make target `../extra/stdout/.deps/libpd_la-stdout.Plo'.  Stop.
make[2]: Leaving directory `/home/inari/pd-0.53-0/src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/inari/pd-0.53-0'
make: *** [all] Error 2
```

It appears the naming scheme for Plo files was changed after 0.51-4 and the extras directory was not updated:
```
[inari@localhost pd-0.53-0]$ cd extra

[inari@localhost extra]$ cd bob~

[inari@localhost bob~]$ ls

bob~.c  bob~-help.pd  GNUmakefile  GNUmakefile.am  GNUmakefile.in  makefile  README.txt
[inari@localhost bob~]$ cd .deps

[inari@localhost .deps]$ ls
bob~.Plo

```

**Relevant versions:**
Linux: CentOS Linux release 7.7.1908 (Core)
GCC: cc version 4.8.5 20150623 (Red Hat 4.8.5-39) (GCC)
Automake: automake (GNU automake) 1.13.4
Make: 3.82

**Comment by fullmetalcoding** [comment]
Amplifying info: 

In 0.53-0, the build system appears to be attempting to creating/copying the extras to src/, but due to a bug it creates the directory src/$(top_srcdir):

`
[inari@localhost pd-0.53-0]$ ls src/
CHANGELOG.txt       d_soundfile_caf.c   g_mycanvas.c   Makefile.in     s_audio_alsa.c       s_libpdmidi.c   u_pdreceive.c     x_time.c
d_arithmetic.c      d_soundfile.h       g_numbox.c     makefile.mac    s_audio_alsa.h       s_loader.c      u_pdsend.c        x_vexp.c
d_array.c           d_soundfile_next.c  g_radio.c      makefile.mingw  s_audio_alsamm.c     s_main.c        x_acoustics.c     x_vexp_fun.c
d_ctl.c             d_soundfile_wave.c  g_readwrite.c  makefile.msvc   s_audio_audiounit.c  s_midi_alsa.c   x_arithmetic.c    x_vexp.h
d_dac.c             d_ugen.c            g_rtext.c      m_atom.c        s_audio.c            s_midi.c        x_array.c         x_vexp_if.c
d_delay.c           g_all_guis.c        g_scalar.c     m_binbuf.c      s_audio_dummy.c      s_midi_dummy.c  x_connective.c    z_hooks.c
d_fft.c             g_all_guis.h        g_slider.c     m_class.c       s_audio_esd.c        s_midi_oss.c    x_file.c          z_hooks.h
d_fft_fftsg.c       g_array.c           g_template.c   m_conf.c        s_audio_jack.c       s_midi_pm.c     x_gui.c           z_libpd.c
d_fft_fftw.c        g_bang.c            g_text.c       m_glob.c        s_audio_mmio.c       s_net.c         x_interface.c     z_libpd.h
d_filter.c          g_canvas.c          g_toggle.c     m_imp.h         s_audio_oss.c        s_net.h         x_libpdreceive.c  z_print_util.c
d_global.c          g_canvas.h          g_traversal.c  m_memory.c      s_audio_pa.c         s_path.c        x_libpdreceive.h  z_print_util.h
d_math.c            g_clone.c           g_undo.c       m_obj.c         s_audio_paring.c     s_print.c       x_list.c          z_queued.c
d_misc.c            g_editor.c          g_undo.h       m_pd.c          s_audio_paring.h     s_stuff.h       x_midi.c          z_queued.h
d_osc.c             g_editor_extras.c   g_vumeter.c    m_pd.h          s_entry.c            s_utf8.c        x_misc.c          z_ringbuffer.c
d_resample.c        g_graph.c           Makefile       m_sched.c       s_file.c             s_utf8.h        x_net.c           z_ringbuffer.h
d_soundfile_aiff.c  g_guiconnect.c      Makefile.am    notes.txt       s_inter.c            s_watchdog.c    x_scalar.c
d_soundfile.c       g_io.c              makefile.gnu   pd.rc           s_inter_gui.c        $(top_srcdir)   x_text.c

`

**Comment by fullmetalcoding** [comment]
Root cause has been identified:
The version of Automake on RHEL7 is too old. configure is supposed to check for this and disable dependency checking, but the check on line 15 of configure.ac does not work. Running configure with dependency checking disabled manually allows PD to build.

**Comment by umlaeute** [comment]
which version of autotools is shipped with RHEL7?

**Comment by umlaeute** [comment]
scratch my last comment, you already provided that info.

**Comment by umlaeute** [comment]
could you check the current `develop` branch (which contains a supposed fix for this in [d046df9](https://github.com/pure-data/pure-data/commit/d046df9fe4be47530f86f02fc32729cc0ea80856)).

**Comment by fullmetalcoding** [comment]
> could you check the current `develop` branch (which contains a supposed fix for this in [d046df9](https://github.com/pure-data/pure-data/commit/d046df9fe4be47530f86f02fc32729cc0ea80856)).

I just tested the fix from `develop` and can confirm it works.
